bin/main.cpp

```diff
@@ -45,8 +45,8 @@ int main(int argc, char** argv)
       
     }
     if (focused) {
-        float deltaS = clock.restart().asSeconds();
-        deltaS = (deltaS < 1.f) ? 1.f : deltaS;
+        float deltaS = clock.restart().asMilliseconds();
+        deltaS = (deltaS > 1.f) ? 1.f : deltaS;
```

Comment:
I thought we agreed we were gonna leave this as seconds, because milliseconds screws everything up

Reply:
I can't remember, this was just what was working last when I committed it.

Reply:
I changed it back to seconds